connector/connect/common/src/test/resources/query-tests/queries/column_when_otherwise.proto.bin

```diff
@@ -0,0 +1,14 @@
+�
+�Z��struct<id:bigint,a:int,b:double,d:struct<id:bigint,a:int,b:double>,e:array<int>,f:map<string,struct<id:bigint,a:int,b:double>>,g:string>OM
```

Comment:
What are these unrecognized characters?

Reply:
it is a binary file. It is not supposed to be readable.